docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.3"
+
+services:
+  mariadb3:
+    image: lscr.io/linuxserver/mariadb:latest
+    container_name: mariadb3
+    environment:
+      - PUID=501
+      - PGID=20
+      - MYSQL_ROOT_PASSWORD=password
+      - TZ=Europe/London
+
+    volumes:
+      - ./.db:/config
+    ports:
+      - 5306:3306
+    restart: unless-stopped
+
+  #baseball:
+  #      image: ubuntu
+  #      restart: unless-stopped
+  #      volumes:
+  #        - "./stuff:/app/stuff"
+  #      command: sleep 999999999
+
+  baseball:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: baseball
+    environment:
+      PORT: "8080"
+    restart: unless-stopped
+    volumes:
+      - "./stuff:/app/stuff"
```

Comment:
After making the changes I mentioned above, I was able to do docker-compose build and it worked. I am not sure where the files being created are going but the image was exported and written.